src/core/lib/channel/channel_args.h

```diff
@@ -160,6 +167,8 @@ class ChannelArgs {
   T* GetPointer(absl::string_view name) const {
     return static_cast<T*>(GetVoidPointer(name));
   }
+  absl::optional<Duration> GetDurationFromIntMillis(
```

Comment:
Will this function be used for filters other than client idle filter?

Reply:
Yeah, all over the place. There'll likely be a `GetDurationFromIntSeconds` too.

ClientIdleFilter/MaxAgeFilter do this a bunch, as do chttp2_transport, and I think there's a few in the TCP code too.